### PR TITLE
FS-3210 Changing Redis service name

### PR DESCRIPTION
**Pull request by Tiny49**
Change description
-Changing the Redis Cluster service name and description to reflect something generic.

 Unit tests and other appropriate tests added or updated
 README and other documentation has been updated / added (if needed)
 Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

How to test
-Will check following deployment

Screenshots of UI changes (if applicable)

### CODE REVIEW

File: apps/pre-award/copilot/environments/addons/funding-service-magic-links.yml [hunk 1]
```diff
@@ -68,12 +68,12 @@ Resources:
         IncludeSpace: false
         PasswordLength: 16
 
-  # Creation of the cluster itself
+  # Creation of the cluster itself (Funding Service Magic Links)
```
Comment (srh-sloan) [inline apps/pre-award/copilot/environments/addons/funding-service-magic-links.yml:71]: Can we change the comment as we've changed the name in the code?

**Review by srh-sloan — CHANGES_REQUESTED** [review]
Small comment, and are there tickets to cover updating the config in the code to use the new name? https://github.com/communitiesuk/funding-service-design-authenticator/blob/main/config/envs/production.py#L14

File: apps/pre-award/copilot/environments/addons/funding-service-magic-links.yml (hunk 1 above)
Comment (Tiny49) [reply apps/pre-award/copilot/environments/addons/funding-service-magic-links.yml:71]: I have created branches for the service changes as well. I will raise the PR now

**Comment by samuelhwilliams** [comment]
Are we still planning to get this in or should we close this PR?

**Pull request closed without merging**